scripts/build-webpack.js

```diff
@@ -58,7 +59,7 @@ const compilerLegacy = webpack({
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
-        query: babelRc.env['pre-node5'],
+        query: babelRc.env.node4,
```

Comment:
@kaylieEB can you open the issue? Because I don't fully understand your comment, I am afraid to write something wrong